### ISSUE TITLE
basic question: Does restart file include genome object information?

### ISSUE THREAD

**Issue by mikegilchrist** [acope3/RibModelFramework#341]
Docs currently indicate that if a restart file is used, all other arguments are ignored.  Is this really true for the genome object argument?

**Comment by acope3** [comment]
No. The only reason we pass genome object into the parameter object is so we get the number of genes. This number is determined in the restart files by the number of \phi values listed.
